front/app/containers/Admin/projects/project/participationContext/utils/votingMethodUtils.tsx

```diff
@@ -1,37 +0,0 @@
-import React, { ReactNode } from 'react';
-import BudgetingInputs from '../components/inputs/VotingInputs/votingMethodInputs/BudgetingInputs';
-import { VotingInputsProps } from '../components/inputs/VotingInputs';
-
-/*
-  Configuration Descriptions:
-  - getVotingMethodInputs: renders specific voting method inputs in back office settings
-  */
-
-export type VotingMethodType = 'budgeting';
-
-export type VotingMethodConfig = {
```

Comment:
@luucvanderzee Why did we remove the new VotingMethodConfig? :thinking: When we implement the new Multiple Voting + Single Voting methods, we'll want a way to easily render method-specific components (without lots of conditionals) right?